lib/openstax/rescue_from/wrapped_exception.rb

```diff
@@ -0,0 +1,76 @@
+module OpenStax
+  module RescueFrom
+    class WrappedException
+      attr_reader :exception
+
+      def initialize(exception)
+        @exception = exception
+      end
+
+      def header
+        @header ||= cause.blank? ? 'An exception occurred' : 'Exception cause'
```

Comment:
Should be a re-written way of achieving this same effect unless I'm completely mistaken

https://github.com/openstax/tutor-server/blob/master/config/initializers/controllers.rb#L71

if there's a cause, header is exception cause, if not a cause, then it's 'an exception occurred'